vmdb/app/models/ems_kubernetes.rb

```diff
@@ -11,6 +14,10 @@ def self.description
     @description ||= "Kubernetes".freeze
   end
 
+  def self.event_monitor_class
```

Comment:
I just moved it up from below.  I was mixed in with other "connection" related methods and didn't belong with the others.